docker/run-test

```diff
@@ -11,11 +11,39 @@ if [[ ${CI_FS_MTR} == 'yes' ]]; then
     CI_FS_DOCKER_FLAG="--mount type=bind,source=/mnt/ci_disk_dir_${CMAKE_BUILD_TYPE},destination=/tmp/ci_disk_dir"
 fi
 
+if [[ "${KEYRING_VAULT_MTR}" == 'yes' ]] && [[ ${DOCKER_OS} != 'centos:6' ]] && [[ ${DOCKER_OS} != 'i386/centos:6' ]]; then
+    if [[ $(docker network list | grep bridge-vault) != *bridge-vault* ]]; then
+        docker network create bridge-vault
+    fi
+
+    # Bootstrap dev mode vault instances
+    BOOTSTRAP_DEV_V1="$(bash $SCRIPTS_DIR/bootstrap-vault --mode=dev --version="0.9.6" --alias=v1 --port="9200" --devtoken=${VAULT_V1_DEV_TOKEN})"
```

Comment:
Please, extract `0.9.6` and `1.5.4` into bash script variables as it used to be in the previous implementation.
Also change `1.5.4` to `1.6.2` as it is the newest version available.

Reply:
Fixed